tools/launch.ts

```diff
@@ -253,26 +315,54 @@ function start() {
   let launchConfig = launchTemplate;
   launchConfig.relaychain.bin = relayBinary;
   launchConfig.relaychain.chain = relayChain;
-  launchConfig.parachains[0].bin = parachainBinary;
-  launchConfig.parachains[0].chain = parachainChain;
 
-  launchConfig.parachains[0].id = argv["parachain-id"] || 1000;
+  let relay_nodes = [];
 
-  launchConfig.relaychain.nodes[0].port = startingPort;
-  launchConfig.relaychain.nodes[0].rpcPort = startingPort + 1;
-  launchConfig.relaychain.nodes[0].wsPort = startingPort + 2;
+  // We need to build the configuration for each of the paras
+  for (let i = 0; i < parachainBinaries.length; i++) {
+    let relayNodeConfig = JSON.parse(JSON.stringify(relayNodeTemplate));
```

Comment:
Why do you need to parse/stringify ?
(This usually leads to some data conversation)